graphene_linked_events/tests/test_api.py

```diff
@@ -647,6 +647,9 @@ def test_pevent_preadded_with_test_events_p_event_relations():
         ("keywordNot", "keyword!"),
         ("allOngoingAnd", "all_ongoing_AND"),
         ("allOngoingOr", "all_ongoing_OR"),
+        ("keywordOrSet1", "keyword_OR_set1"),
+        ("keywordOrSet2", "keyword_OR_set2"),
+        ("keywordOrSet3", "keyword_OR_set3"),
     ],
 )
 def test_resolve_events_unsupported_parameter_mapping(
```

Comment:
This was not changed in this PR but just curious why is this function named like it is? Unsupported parameter mapping? Why unsupported? Isn't it clearly supported as they're being tested that the mapping happens in a certain way? Maybe the naming of this function could be made better to signify what's going on here, it is unclear IMHO.

Reply:
I think it tests something like, if you would add custom parameters to the URL, it would not break the LinkedEvents query.